doc/cookbook/source/examples/classifier/linear_svm.rst

```diff
@@ -0,0 +1,60 @@
+=============================
+Linear Support Vector Machine
+=============================
+
+Linear Support Vector Machine is a binary classifier which finds a hyper-plane such that the margins between the two classes are maximized. These margins can be described by the following equations:
+
+.. math::
+    {\bf w}^\top{\bf x} + b = 1
+
+    {\bf w}^\top{\bf x} + b = -1
+
+where :math:`{\bf w}` is vector of weights, :math:`{\bf x}` is feature vector and :math:`b` is bias.
+
+The distance between these two margins is :math:`\frac{2}{\|\mathbf{w}\|}`, so we want to minimize :math:`\|\mathbf{w}\|`. By using Lagrange Multipliers, we can show the solution to be:
```

Comment:
Avoid Lagrange multipliers. Keep in mind this is not to explain the algorithm or math, but just show how to use the API in Shogun. You can say "The solution takes the form ...."